### ISSUE TITLE
Cannot integrate with rubocop, overcommit

### ISSUE THREAD

**Issue by jamesinfmz** [sds/slim-lint#9]
I've been trying to integrate slim-lint with rubocop and overcommit this morning, and have yet to be able to have it successfully run either rubocop is run, or when a commit is made. The instructions given in README are somewhat sparse. What I've tried so far:

- Adding the following to `.overcommit.yml`:
```
SlimLint:
    enabled: true
    on_warn: fail # Treat all warnings as failures
```

- Adding the following to `.rubocop.yml`:
```
SlimLint:
  enabled: true
```

- Adding `require: slim_lint` to `rubocop.yml`

What is the current mechanism for integrating slim-lint with rubocop?

**Comment by sds** [comment]
Hey @jamesinfmz,

The following works for me:

```yaml
# .overcommit.yml in repository root
SlimLint:
    enabled: true
    on_warn: fail # Treat all warnings as failures
```

Ensure `slim-lint` is installed:
```bash
gem install slim_lint
```

Install Overcommit in your repository:
```bash
gem install overcommit && cd path/to/my/repo && overcommit --install
```

Attempt to commit a file with a problem:
```slim
# test.slim
- unused_variable = 'Unused!'
```
```bash
git add test.slim
git commit
```

I see the following output:
```
Running pre-commit hooks
Analyzing with slim-lint...................................[SlimLint] WARNING
Warnings on modified lines:
/home/shane/test/test.slim:1 [W] RuboCop: Useless assignment to variable - `unused_variable`.


⚠ All pre-commit hooks passed, but with warnings
```

I'm not sure where you got the idea that your `.rubocop.yml` file need to have `SlimLint` added to it. It is not a cop, it is a separate tool that invokes RuboCop under the hood.

Hope this helps.

**Comment by jamesinfmz** [comment]
Thanks. Turns out that there was a dependency on .overcommit 0.25.0; the version that `gem install overcommit` installed for me today was 0.24.0.
